### PR TITLE
[WorldMoodTracker] Initialized World Mood Tracker app

### LINKED ISSUE #188: [WorldMoodTracker] Add initial app
### Short Description

Since the app will be developed using React, the bundled packages are also needed to be added to git.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixes #188.

**Changes:**

* Add Webpack, babel and related configurations
* Ignore node_modules for sub apps
* Add missing glyohicons font
* Ignore dynamic files for coala

**Screenshots for the change:**

screenshot from 2017-06-16 13 56 23

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by singhpratyush** [comment]
@daminisatya @djmgit @kavithaenair: Please review.

File: .coafile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [default]
 bears = LineLengthBear, SpaceConsistencyBear
 files = **.js, **.css, **.html, **.yml, **.json
-ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js
+ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js, **/node_modules/**, **/index_bundle.js, WorldMoodTracker/index.html
```
Comment (kavithaenair) [inline .coafile]: index.html and js files written by you should not be ignored by coala bears i guess.

File: .coafile [hunk 2]
```diff
@@ -20,7 +20,7 @@ use_spaces = True
 [js]
 bears = PHPCodeSnifferBear
 files = **.js
-ignore = **.min.js, tweetheatmap/public/lib/**, /app/webtweets/js/bootstrap/**
+ignore = **.min.js, tweetheatmap/public/lib/**, /app/webtweets/js/bootstrap/**, **/node_modules/**, **/index_bundle.js
```
Comment (kavithaenair) [inline .coafile]: Here as well.

File: .coafile (hunk 1 above)
Comment (singhpratyush) [inline .coafile]: It is not like that. The two files ignored are generated by `webpack` while the build process.
```sh
cd WorldMoodTracker
yarn build
```

Comment (kavithaenair) [inline .coafile]: Oh. Okay! 👍

Comment (singhpratyush) [inline .coafile]: See [here](https://github.com/fossasia/apps.loklak.org/pull/189/files#diff-675bbc883c350abe040e9b7515e8a8b4R15) and [here](https://github.com/fossasia/apps.loklak.org/pull/189/files#diff-675bbc883c350abe040e9b7515e8a8b4R6).

**Comment by singhpratyush** [comment]
@Achint08 @hemantjadon: Please take a look.

**Comment by mariobehling** [comment]
Great to see this getting started! Thank you!

**Pull request merged**